build/lint.mk

```diff
@@ -0,0 +1,38 @@
+################################################################################
+###                             Required Variables                           ###
+################################################################################
+ifndef DOCKER
+$(error DOCKER not set)
+endif
+
+ifndef BUILD_CACHE_DIR
+$(error BUILD_CACHE_DIR not set)
+endif
+
```

Comment:
```
ifeq ($(shell which $(DOCKER_CMD) >/dev/null 2>&1; echo $$?), 1)
$(error "Docker command '$(DOCKER_CMD)' not found. Please install Docker.")
endif
```

Reply:
Good suggestion, but does feel outside the scope of the lint include.  I opened a separate PR at https://github.com/Kava-Labs/kava/pull/1992 and added the check with some simplifications to where DOCKER is defined.